fe/fe-core/src/main/java/org/apache/doris/analysis/OutFileClause.java

```diff
@@ -216,6 +250,43 @@ private void getBrokerProperties(Set<String> processedPropKeys) {
         brokerDesc = new BrokerDesc(brokerName, brokerProps);
     }
 
+    private void getParquetProperties(Set<String> processedPropKeys) throws AnalysisException {
```

Comment:
Better add comment with an example to explain what kind of property format does this method parsed